### PR TITLE
Added Adminer to compose

### PR DESCRIPTION
**Pull request by gintsgints**
- Added adminer image for fast/simple database tool.
- Added bit instructions on README.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -68,6 +68,11 @@ services:
     ports:
       - "27017:27017"
 
+  adminer:
+    image: adminer
+    restart: always
+    ports:
+      - 8080:8080
```
Comment (pleerock) [inline docker-compose.yml]: we need to use different port something that isn't busy for users for sure

**Comment by pleerock** [comment]
okay I checked adminer and tbh I don't like it. Isn't there modern alternatives? If there isn't I think lets just let users to use what they wanna use and do not include some specific tools in docker compose.

**Comment by gintsgints** [comment]
Ok. I remove adminer then.

**Comment by pleerock** [comment]
sorry about that 😞 Thanks for the contribution!

**Pull request closed without merging**